docs/install.rst

```diff
@@ -166,16 +157,16 @@ Numpy in order to build Astropy.
 
 You will also need `Cython <http://cython.org/>`_ (v0.21 or later) and
 `jinja2 <http://jinja.pocoo.org/docs/dev/>`_ (v2.7 or later) installed
-to build from source, unless you are installing a numbered release. (The
-releases packages have the necessary C files packaged with them, and hence do
+to build from source, unless you are installing a stable release. (The
```

Comment:
This would imply to me "stable" in the way we define it: only the most recent release.  By "numbered" I think it more generally means "not dev".  Perhaps the clearest thing is just "installing a release", though?

Reply:
Yes sounds good!